company/how-to-guides-for-staff/how-to-purchase/how-to-arrange-a-bounty-program.md

```diff
@@ -14,11 +14,11 @@ When a bounty project has been identified, the bounty lead at Mattermost will pr
 * Start and end date of the bounty program project
 * Estimated bounty to be paid
 
-The document will be sent to the [Bounty MM Channel](https://community-release.mattermost.com/private-core/channels/developer-bounty-program) where Finance will review the spend. If spending is approved, you may begin to engage the community and prospects.
+The document will be sent to the [Bounty MM Channel](https://community.mattermost.com/private-core/channels/developer-bounty-program) where Finance will review the spend. If spending is approved, you may begin to engage the community and prospects.
 
 ## Countries/regions not allowed at this time
 
-Although we’d like to be able to administer a Bounty Program to anyone and anywhere in the world, we cannot. Similar to the no hire list, all bounty programs are governed by the **\[Location Strategy List\].** If you are engaging in a Bounty Program, please reach out to [natalie.jew@mattermost.com](mailto:natalie.jew@mattermost.com) and [tim@mattermost.com](mailto:tim@mattermost.com) prior to starting any work to ensure that person resides in a country/region we can support. Please do not engage someone in the community to partake in a Mattermost sponsored bounty program if we do not support business dealings in the country/region as we will be unable to pay the contributor.
+Although we’d like to be able to administer a Bounty Program to anyone and anywhere in the world, we cannot. Similar to the no hire list, all Bounty Programs are governed by the **\[Location Strategy List\].** If you are engaging in a Bounty Program, please reach out to [natalie.jew@mattermost.com](mailto:natalie.jew@mattermost.com) and [tim@mattermost.com](mailto:tim@mattermost.com) prior to starting any work to ensure that person resides in a country/region we can support. Please do not engage someone in the community to participate in a Mattermost-sponsored Bounty Program if we do not support business dealings in the country/region as we will be unable to pay the contributor.
```

Comment:
There isn't a link to the Location Strategy List.

Reply:
@justinegeffen confirmed with Natalie that we will not be posting the location doc at this time. We could just remove this as placeholder link

Reply:
Thanks, @TQuock!